report/renderer/i18n.js

```diff
@@ -32,36 +32,42 @@ export class I18n {
 
   /**
    * @param {number} number
-   * @param {number} granularity
+   * @param {number|undefined} granularity
    * @param {Intl.NumberFormatOptions} opts
```

Comment:
Bump on this if it's not too much work.

Reply:
just now: I added `=` to the opts type to better reflect how this method is used.

granularity should remain `undefined` because all the use cases here are passing a value explicitly, even if it may be undefined.

all this stuff is pretty subjective i think. if you feel strongly I'll make the change.

Reply:
Yeah that's fine with me